### PR TITLE
adding strain name info to sketch_service

### PR DESCRIPTION
**Pull request by slebras**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -21,6 +21,7 @@ RUN pip install --upgrade pip && \
     pip install --upgrade --extra-index-url https://pypi.anaconda.org/kbase/simple \
       -r requirements.txt \
       kbase-workspace-utils==0.0.13 && \
+      pandas && \
```
Comment (jayrbolton) [inline Dockerfile]: lock this to at least a minor version, or a full version

File: src/utils/map_strains.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import pandas as pd
+
+
+def map_strains(distances):
+    # TODO insert strain names here
```
Comment (jayrbolton) [inline src/utils/map_strains.py]: remove comment

File: src/utils/map_strains.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    print('distances', distances)
+    df = pd.read_csv('data/gcf_strain_map.csv')
+    refseq_ids = [d['sourceid'] for d in distances]
+    df = df[df['refseq_id'].isin(reseq_ids)]
```
Comment (jayrbolton) [inline src/utils/map_strains.py]: typo here -- should this be`refseq_ids`, not `reseq_ids`

File: src/utils/perform_search.py [hunk 4]
```diff
@@ -18,8 +19,9 @@ def perform_search(sketch_path, db_name, max_results=10):
     print('search done in', time.time() - start_time)
     if response.status_code == 200:
         resp_json = response.json()
-        # Convert Refseq IDs into KBase IDs
+        # Convert Refseq IDs into KBase IDs (only does these if we are in the refseq namespace)
```
Comment (jayrbolton) [inline src/utils/perform_search.py]: is there somewhere where we check which namespace we're in before doing "map_strains"? I dont see this

Comment (slebras) [reply src/utils/perform_search.py]: right now no, but the way they work it shouldn't fail, but I'll add that check for good measure

File: Dockerfile (hunk 1 above)
Comment (slebras) [reply Dockerfile]: ive changed this to be a separate pip install outside of the pip install --upgrade but still in the same RUN statement

**Comment by jayrbolton** [comment]
Looks bueno, i will merge and test on my laptop

**Pull request merged**